deployment/network-operator/templates/operator.yaml

```diff
@@ -74,6 +74,14 @@ spec:
             - name: HTTPS_PROXY
               value: {{ .Values.proxy.httpsProxy }}
             {{- end }}
+            {{- if .Values.proxy.httpProxy | empty | not }}
+            - name: http_proxy
+              value: {{ .Values.proxy.httpProxy }}
+            {{- end }}
+            {{- if .Values.proxy.httpsProxy | empty | not }}
+            - name: https_proxy
+              value: {{ .Values.proxy.httpsProxy }}
+            {{- end }}
             {{- if .Values.proxy.noProxy | empty | not }}
             - name: NO_PROXY
```

Comment:
Do we want to add no_proxy in lowercase as well? This is mostly for consistency, I don't know if any tool that strictly require lowercase version exist.